my_compassion/static/src/css/my2_login.css

```diff
@@ -0,0 +1,174 @@
+/*
+    This stylesheet concerns the my2_login_template component
+    templates/my2_login_template.xml
+*/
+
+/* Header container styling */
+.header-elements {
+  position: relative;
+  width: 100%;
+  padding: 1rem;
+  margin-bottom: 20rem;
+}
+
+/* Main login box container */
+.my2-login-box {
+  background-color: #ffffff;
+  padding: 2rem;
+  max-width: 380px;
+  width: 100%;
+}
+
+/* Logo styling */
+.logo {
+  height: 100px;
+  position: relative;
+  left: 15%;
+  transform: translateX(-50%);
+}
+
+/* Circular icon styling */
+.my2-login-box .icon {
+  height: 200px;
+  clip-path: ellipse(50% 50% at 50% 50%);
+}
+
+/* Hide logo on mobile devices */
+@media (max-width: 767px) {
+  .my2-login-box .logo {
+    display: none;
+  }
+}
+
+/* Welcome message styling */
+.my2-login-box .welcome {
+  font-weight: 600;
+  font-size: 1.25rem;
+  color: #005eb8;
+  margin-bottom: 1.5rem;
+}
+
+/* Form input field styling */
+.my2-login-form .form-control {
+  border-radius: 0.5rem;
+  padding: 0.75rem;
+  font-size: 0.95rem;
+}
+
+/* Form group spacing */
+.my2-login-form .form-group {
+  margin-bottom: 1rem;
+}
+
+/* Password toggle button styling */
+.toggle-password {
+  position: absolute;
+  right: 1rem;
+  top: 0;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  color: #999;
+}
+
+/* Primary button styling */
+.btn-primary {
+  background-color: #005eb8;
+  border-color: #005eb8;
+  border-radius: 0.5rem;
+  padding: 0.6rem 1rem;
+  font-weight: 500;
+  font-size: 1rem;
+}
+
+/* Link styling within login box */
+.my2-login-box a {
+  text-decoration: underline;
+}
+
+/* Button hover effect */
+.btn-primary:hover {
+  background-color: #004a94;
+}
+
+/* Forgot password link styling */
+.forgot-password {
+  font-size: 1rem;
+  color: #005eb8;
+  text-decoration: none;
+  text-align: left !important;
+  margin-left: 0;
+}
+
+/* Forgot password hover effect */
+.forgot-password:hover {
+  text-decoration: underline;
+}
+
+/* Mobile responsiveness for login box */
+@media (max-width: 768px) {
+  .my2-login-box {
+    max-width: 100%;
+  }
+}
+
+/* Language picker container styling */
+.language-picker {
+  padding-top: 20rem;
+  min-height: calc(100vh - 300px);
+  position: absolute;
+  right: 15%;
+  transform: translateX(50%);
+}
+
+/* Language picker wrapper styling */
+.language-picker-wrapper {
+  position: relative;
+  display: inline-block;
+}
```

Comment:
This seems related to the menu, it should be place else where then my2_login.css and be part of another PR
```suggestion
```